transform/mattermost-analytics/models/intermediate/product/configs/int__config_ldap.sql

```diff
@@ -0,0 +1,101 @@
+{{
+    config({
+        "materialized": "table",
+        "snowflake_warehouse": "transform_xs"
```

Comment:
These models took significant time to run on xs warehouse. Perhaps we can test them on a large. However we'll need a clean environment in order to avoid running them in incremental mode.